CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 6.4.0
+
+### Added 
+* Deployment/CLI: OneFuzz Config refactored - `authority`, `tenant_domain`, and `cli_client_id` are now required values in the config.json used during deployment [#2771](https://github.com/microsoft/onefuzz/pull/2771)
```

Comment:
```suggestion
* Deployment/CLI: OneFuzz Config refactored - `tenant_id`, `tenant_domain`, `multi_tenant_domain`, and `cli_client_id` are now required values in the config.json used during deployment and no longer required at when running the config command. [#2771](https://github.com/microsoft/onefuzz/pull/2771)
```

Reply:
You might want to remove the `at` in `no longer required at`

Reply:
updated.